goals.md

```diff
@@ -0,0 +1,84 @@
+# End User Working Group
+## Mission and Goals
```

Comment:
Headers should have a trailing new line, or it might get flagged during markdown linting.